[Follow-up] Fix CI test failures in API controller tests

## 概要

PR #140 での Issue #139 の実装中に、CI環境で他のAPIコントローラーテストが失敗していることが発見されました。これらのテストはローカルでは問題なく、Issue #139の変更とは直接関係ありませんが、CI/CDパイプラインの安定性のために修正が必要です。

## 背景

Issue #139でjournal entries controllerのテストを修正中、CI環境で以下のコントローラーテストが失敗していることが判明しました：
- organizations.controller.test.ts
- accountingPeriods.controller.test.ts  
- auth.controller.test.ts
- reports.controller.test.ts
- accounts.controller.test.ts
- ledgers.controller.test.ts
- auditLog.controller.test.ts (TypeScript error)

## 問題の詳細

### 1. TypeScriptエラー
`auditLog.controller.test.ts` で 'details' フィールドがPrismaスキーマに存在しないエラー：
```
error TS2353: Object literal may only specify known properties, and 'details' does not exist in type
```

### 2. HTTPステータスコードの不一致
複数のテストで期待されるステータスコードと実際のレスポンスが異なる：
- Expected 403, but received 200
- Expected 404, but received 200
- Expected 400, but received 201

## 推奨される解決方法

1. **auditLog.controller.test.ts の修正**
   - Prismaスキーマを確認し、'details' フィールドの有無を確認
   - テストデータを正しいスキーマに合わせて修正

2. **各コントローラーテストの検証**
   - 期待されるステータスコードが正しいか確認
   - 実際のコントローラーの動作と一致させる
   - 必要に応じてテストまたはコントローラーを修正

3. **CI環境とローカル環境の差異調査**
   - 環境変数の違いを確認
   - データベースの初期状態の違いを確認

## 技術的考慮事項

- CI環境でのみ発生する問題の可能性
- テストの実行順序による影響の可能性
- データベースのシード/クリーンアップの問題の可能性

## 影響範囲

- 影響を受けるコンポーネント: APIコントローラーテスト全般
- ユーザーへの影響: なし（テストのみ）
- 優先度: Medium（CI/CDの安定性に影響）

## 受け入れ条件

- [ ] すべてのAPIコントローラーテストがCI環境で成功する
- [ ] TypeScriptエラーが解消される
- [ ] ローカル環境とCI環境の両方でテストが通る
- [ ] 既存の機能に影響を与えない

## 関連リンク

- 元のIssue: #139
- 実装PR: #140
- CI失敗ログ: https://github.com/knishioka/simple-bookkeeping/actions/runs/16910347379

## テストコマンド

```bash
# ローカルでCI環境を再現
NODE_ENV=test pnpm test

# 個別のテスト実行
NODE_ENV=test pnpm --filter @simple-bookkeeping/api test -- organizations.controller.test.ts
NODE_ENV=test pnpm --filter @simple-bookkeeping/api test -- auditLog.controller.test.ts
```

--- a/apps/api/src/controllers/__tests__/auditLog.controller.test.ts
+++ b/apps/api/src/controllers/__tests__/auditLog.controller.test.ts
@@ -33,7 +33,7 @@ describe('AuditLogController', () => {
         action: 'CREATE',
         entityType: 'JournalEntry',
         entityId: 'test-entity-1',
-        details: { description: 'Created journal entry' },
+        newValues: { description: 'Created journal entry' },
         ipAddress: '127.0.0.1',
       },
     });
@@ -45,7 +45,8 @@ describe('AuditLogController', () => {
         action: 'UPDATE',
         entityType: 'Account',
         entityId: 'test-entity-2',
-        details: { changes: { name: 'Updated name' } },
+        oldValues: { name: 'Original name' },
+        newValues: { name: 'Updated name' },
         ipAddress: '127.0.0.1',
       },
     });
@@ -100,8 +101,8 @@ describe('AuditLogController', () => {
         .get('/api/v1/audit-logs')
         .set('Authorization', `Bearer ${adminToken}`)
         .query({
-          from: today.toISOString().split('T')[0],
-          to: tomorrow.toISOString().split('T')[0],
+          startDate: today.toISOString(),
+          endDate: tomorrow.toISOString(),
         });
 
       expect(response.status).toBe(200);
@@ -148,7 +149,7 @@ describe('AuditLogController', () => {
 
       expect(response.status).toBe(200);
       expect(response.body.data.id).toBe(log?.id);
-      expect(response.body.data).toHaveProperty('details');
+      expect(response.body.data).toHaveProperty('newValues');
     });
 
     it('should return 404 for non-existent log', async () => {
@@ -252,7 +253,11 @@ describe('AuditLogController', () => {
       });
 
       expect(auditLog).not.toBeNull();
-      expect(auditLog?.details).toHaveProperty('description', 'Test Entry for Audit');
+      expect(auditLog?.newValues).toEqual(
+        expect.objectContaining({
+          description: 'Test Entry for Audit',
+        })
+      );
     });
 
     it('should create audit log on account update', async () => {

--- a/apps/api/src/test-utils/test-helpers.ts
+++ b/apps/api/src/test-utils/test-helpers.ts
@@ -13,6 +13,7 @@ export const cleanupTestData = async () => {
   // Delete in correct order to respect foreign key constraints
   await prisma.journalEntryLine.deleteMany({});
   await prisma.journalEntry.deleteMany({});
+  await prisma.auditLog.deleteMany({});
   await prisma.account.deleteMany({});
   await prisma.accountingPeriod.deleteMany({});
   await prisma.userOrganization.deleteMany({});

